freeplane/src/main/java/org/freeplane/main/application/MapViewDockingWindows.java

```diff
@@ -208,9 +209,37 @@ public void internalFrameClosing(InternalFrameEvent e) {
             }
 		};
 
+		addTabsPopupMenu(rootWindow);
 
 	}
 
+	private void addTabsPopupMenu(DockingWindow dockingWindow){
+		dockingWindow.setPopupMenuFactory(new WindowPopupMenuFactory() {
+		   public JPopupMenu createPopupMenu(DockingWindow window) {
+			   if(window instanceof ConnectedToMenuView) {
+				   return null;
+			   }
+			   JPopupMenu menu = new JPopupMenu(window.getTitle());
+			   JMenuItem menuItem = new JMenuItem("Rename tab (temporarily)");
```

Comment:
Add translation keys and English versions to /freeplane/src/viewer/resources/translations/Resources_en.properties and use them through TextUtils.getText(String, String). Don't include any texts directly in the code because of translations.